src/github.com/matrix-org/go-neb/realms/jira/urls/urls_test.go

```diff
@@ -0,0 +1,45 @@
+package urls
+
+import (
+	"testing"
+)
+
+var urltests = []struct {
+	in      string
+	outKey  string
+	outBase string
+	outRaw  string
```

Comment:
I found it slightly confusing that these are given in a different order than in the JIRAURL struct definition.